src/Wizard.js

```diff
@@ -116,6 +118,27 @@ export default class Wizard extends Webform {
       }
     });
 
+    this.on('renderRows', (newPage) => {
```

Comment:
This code "feels" like it will break something. Please join dev support so we can talk through what you are doing here.